docs/how-to-test.md

```diff
@@ -31,15 +31,15 @@ To run the unit test, you can simply run the following command:
 ./gradlew test -PskipITs
 ```
 
-This command runs all the unit tests and skip the integration tests.
+This command runs all the unit tests and skips the integration tests.
 
 ## Run the integration tests
 
-Gravitino has two modes to run the integration tests, the `embedded` and `deploy` modes.
+Gravitino has two modes to run the integration tests, the `embedded`(default) and `deploy` modes.
```

Comment:
Need space between embedded and "(default)", but I would change to ".. the default embedded mode and the deploy mode."

Reply:
OK